x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/NodesDeprecationCheckAction.java

```diff
@@ -32,19 +31,13 @@ private NodesDeprecationCheckAction() {
         super(NAME);
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA) // this can be replaced with TransportRequest.Empty in v9
```

Comment:
Slack thread confirming that we don't need it anymore: https://elastic.slack.com/archives/C8UUBNASY/p1733330314431679